recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -0,0 +1,2 @@
+# Append domain name
+hostname += "-Dom0"
```

Comment:
on my board it will look like salvator-x-m3-xt-Dom0
can we make it all lowercase please?

Reply:
Done